src-test/copyright.c

```
@@ -0,0 +1,103 @@
+	int i;
+	char byte[2] = "\0\0";
+
+	for (i = 0; i < (MAX_COPYRIGHT - 1); i++)
+	{
+		*byte = copyright[i];
+		if (!*byte) break;
+		else if (isalnum(*byte)) out[i] = *byte;
+		else if (strstr(" @#^()[]-_+;:.<>",byte)) out[i] = *byte;
+		else out[i] = ' ';
+	}
+	out[i] = 0;
```

Cppcheck (2.19.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)